waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/AbstractMetaStore.java

```diff
@@ -143,6 +144,14 @@ public void setWritableDatabaseWhiteList(List<String> writableDatabaseWhitelist)
     this.writableDatabaseWhitelist = writableDatabaseWhitelist;
   }
 
+  public long getTimeout() {
```

Comment:
This needs to have a better name. Something like, `federatet-client-call-timeout-in-ms` I'm open for better suggestions, it should definitely tell us if the user should specify millis or seconds or minutes etc.. This needs to go in the README as well at some point